addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/parts/ScEnvSshConnRowExpanded.js

```diff
@@ -88,25 +99,73 @@ class ScEnvSshConnRowExpanded extends React.Component {
         <Table.Cell colSpan="3" className="p3">
           <Grid columns={2} stackable>
             <Grid.Row stretched>
-              <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              {this.isAppStreamEnabled ? (
+                <Grid.Column width={12}>{this.renderAppStreamInfo()}</Grid.Column>
+              ) : (
+                <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              )}
               <Grid.Column width={4}>
                 <Segment className="flex items-center">
-                  <div className="w-100 overflow-hidden">{this.renderCountDown()}</div>
+                  <div className="w-100">{this.renderCountDown()}</div>
                 </Segment>
               </Grid.Column>
             </Grid.Row>
           </Grid>
-          <div className="mt3">
-            Example:
-            <Segment className="mt2">
-              {example} <CopyToClipboard text={example} className="ml2 mt0" />
-            </Segment>
-          </div>
+          {this.isAppStreamEnabled ? (
+            <></>
+          ) : (
+            <div className="mt3">
+              Example:
+              <Segment className="mt2">
+                {example} <CopyToClipboard text={example} className="ml2 mt0" />
+              </Segment>
+            </div>
+          )}
         </Table.Cell>
       </Table.Row>
     );
   }
 
+  renderAppStreamInfo() {
+    const interfaces = this.networkInterfaces;
+    const moreThanOne = _.size(interfaces) > 1;
+
+    return (
+      <div>
+        <b>Connection instructions for your AppStream workspace:</b>
+        <List bulleted>
+          <List.Item>
+            The IP Address or DNS of the instance.{' '}
+            {moreThanOne ? 'Ask your administrator if you are not sure which one to use:' : ''}
```

Comment:
We shouldn't have moreThanOne condition true for AppStream for now. We can keep this check around but just wanted to make sure we don't display it currently for any environment when AppStream is enabled..

Reply:
True. Doesn't hurt to have it, but I don't see it getting used with AppStream as well.